docs/contributing.rst

```diff
@@ -1,168 +1,5 @@
-Contributing to GraSPy
-======================
+Contributing to graspologic
+===========================
 
-(adopted from scikit-learn)
+Please see: `Contributing to graspologic <https://github.com/microsoft/graspologic/blob/dev/CONTRIBUTING.md>`_.
```

Comment:
@j1c I know you said you didn't like this, and I do agree, but I did it for now because it was already out of date from what we had in CONTRIBUTING.md.

I'm going to make a new issue to figure out how to get the common things from a different directory other than docs, as markdown, into the documentation generated for the actual site